/RefereeSystem:
  ros__parameters:
    buffersize: 4096
    serialport_arry:
    - /dev/ttyUSB0
    - /dev/ttyUSB1
    - /dev/ttyUSB2
    # - /dev/ttyACM0
    # - /dev/ttyACM1
    # - /dev/ttyACM2
    use_sim_time: false
    file_output: false
    file_output_path: /home/suzuki/RefereeSystem/
    replay_file_path: /home/suzuki/Downloads/dump/serialPortdump_20240515153746.txt
